lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -55,6 +54,12 @@ def self.fabricpath_feature
       return :disabled
     end
 
+    def self.fabricpath_enable
+      # TBD: Move to Feature provider
+      FabricpathGlobal.fabricpath_feature_set(:enabled) unless
+        FabricpathGlobal.fabricpath_feature == :enabled
```

Comment:
Totally agree, hence the  `# TBD: Move to Feature provider` comment :-)

This def I added is just a helper to make the calls cleaner but it's still an interim step. These Fabricpath feature calls are scattered through four different providers so moving it into Feature is outside the scope of my changes, but yes let's refactor it asap.